packages/core/test-project/ios/RNEmbraceTests/RNEmbraceTests.swift

```diff
@@ -799,3 +799,70 @@ class EmbraceSpansSDKNotStartedTests: XCTestCase {
     }
 }
 
+class ComputeBundleIDTests: XCTestCase {
+    // https://nshipster.com/temporary-files/
```

Comment:
Probably many of this tests under the hood, are using `UserDefaults`. `UserDefaults` writes things to disk.
So, to prevent any kind of flakyness, i'd remove all things persisted in UserDefaults between tests. In order to achieve this (considering you've [made this change](https://github.com/embrace-io/embrace-react-native-sdk/pull/72/files#r1731339931), i'd add this code:

```swift
    override func tearDownWithError() throws {
          UserDefaults.standard.removePersistentDomain(forName: "EmbraceReactNative")
    }
```

Reply:
added